.github/ISSUE_TEMPLATE/plugin-request.md

```diff
@@ -0,0 +1,19 @@
+---
+name: Plugin request
+about: Suggest an idea for a new plugin
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- Be aware that MooLite can not make any actions for you, it can only provide QOL plugins -->
```

Comment:
```suggestion
<!-- Be aware that MooLite can not make any actions for you, it can only provide QOL plugins to track game events and update UI -->
```